docs/BEMAN_STANDARD.md

```diff
@@ -112,27 +111,35 @@ to build it, and links to further documentation.
 
 ## `README.md`
 
-**[README.TITLE]** RECOMMENDATION: The `README.md` should begin with a level 1
-header with the name of the library optionally followed with a ":" and short
-description.
+**[README.TITLE]** RECOMMENDATION: The `README.md` must begin with a level 1 header with the name of the library optionally followed with a ":" and short description.
```

Comment:
change `Examples` at line 116 to `Example`, as it is only one example

Reply:
I think this is just the convention in that file.